sql/moz-fx-data-shared-prod/fivetran_costs_derived/daily_connector_costs_v1/query.sql

```diff
@@ -0,0 +1,67 @@
+WITH incremental_mar AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.incremental_mar_v1`
+),
+destinations AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.destinations_v1`
+),
+monthly_costs AS (
+  SELECT
+    *
+  FROM
+    `moz-fx-data-shared-prod.fivetran_costs_derived.monthly_costs_v1`
+),
+monthly_costs_per_mar AS (
+  SELECT
+    destination_id,
+    measured_month,
+    monthly_costs.dollars_spent / NULLIF(SUM(incremental_mar.active_rows), 0) AS cost_per_mar
```

Comment:
`cost_per_mar` could end up being a very small value so I was initially concerned some precision would be lost, but it looks like it ends up as a `BIGNUMERIC` which has 38 decimal places, so it should be fine.